engine/src/main/java/org/terasology/rendering/world/RenderableWorldImpl.java

```diff
@@ -372,7 +381,7 @@ public boolean isChunkValidForRender(RenderableChunk chunk) {
     }
 
     public boolean isChunkVisibleLight(RenderableChunk chunk) {
-        return isChunkVisible(shadowMapCamera, chunk);
+        return isChunkVisible(dag.get(ShadowMapNode.class).getCamera(), chunk); //TODO: find an elegant way
```

Comment:
Hmmm. I have a number of concerns here.
1. dat.get(class) seems to imply there should be only once instance of a given node class. Without thinking too deeply about it I have the impression we might now want to limit it like that. I'd rather work with nodes declaring/registering their own name instead. I.e. in the case of the Blur nodes, BlurForDoF and BlurForBloom for example. That been said, we might want to keep this for a later stage.
2. I am not so fond on the idea of fetching the ShadowMapNode from a map and then fetching the camera from that node, potentially frequently. Perhaps the RenderableWorldImpl could have a setShadowMapCamera() method? Or a setShadowMapNode() which sets the camera?

That been said, let's rename this method to isChunkVisibleFromMainLight(),  which is what this method does. 

Nevermind that at some point we'll want multiple main lights. How do we get Tatooine in Terasology otherwise?? =)